relay/circuit-v2.md

```diff
@@ -294,6 +294,42 @@ Common failure status codes are:
 
 ***Note: implementations _should not_ accept connection initiations over already relayed connections***
 
+##### Security protocol selection for the relayed connection
+
+Instead of negotiating the security protocol in-band, security protocols should
```

Comment:
I think this should be a `may` instead of a `should`.

Reply:
I think it should be a `MAY`, at least for the time being. We can change it to `SHOULD` later, when all is implemented and the spec is ready for candidate recommendation status.